conf/are.conf.part

```diff
@@ -4,7 +4,7 @@
 
 source src_ch_are_landschaftstypen : def_queryable_features
 {
-    sql_db = are
+    sql_db = are_dev
```

Comment:
+1

(Maybe it would make sense to have something like `are_dummy` in the input file. So it would be clear that this would get replaced; but I can live with current state)